cvat/apps/engine/serializers.py

```diff
@@ -715,6 +716,13 @@ class DataSerializer(serializers.ModelSerializer):
         help_text="Uploaded files")
     server_files = ServerFileSerializer(many=True, default=[],
         help_text="Paths to files from a file share mounted on the server, or from a cloud storage")
+    server_files_exclude = serializers.ListField(required=False, default=[],
+        child=serializers.CharField(max_length=1024),
```

Comment:
Doest it mean that path cannot be more than 1024 symbols? Why 1024 exactly?

Reply:
I used the same limit as for server_files